examples/bq_file_load_benchmark/benchmark_tools/file_constants.py

```diff
@@ -0,0 +1,48 @@
+"""
+file_constants.py holds a dict called FILE_CONSTANTS that holds information
+for types of files involved in the benchmark process.
+"""
+
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
```

Comment:
Seems like you don't need this division or print function imports in this file?

Reply:
yeah I don't really need it, but the style guide said to include it in "all new code" : https://google.github.io/styleguide/pyguide.html#2204-decision, so I threw it in haha